packages/blocks/src/api/registration.js

```diff
@@ -191,6 +191,11 @@ export function unregisterBlockType( name ) {
  * @param {string} name Block name.
  */
 export function setUnknownTypeHandlerName( name ) {
+	deprecated( 'setUnknownTypeHandlerName', {
+		plugin: 'Gutenberg',
+		version: '4.0',
```

Comment:
These versions need to be bumped by at least .2 :)

Reply:
Both in this file and in `store/*`, can't we simply treat the deprecated `*UnknownType*` functions as aliases for their `*FreeformContent*` counterparts? If I read the diff correctly, one deprecated function keeps calling a deprecated action creator, which keeps outputting an action of deprecated type.

Reply:
Following up on this one [here](https://github.com/WordPress/gutenberg/pull/8274#discussion_r223131741).

Reply:
> These versions need to be bumped by at least .2 :)

I ended up bumping them to 4.4 because we already have documented deprecations for that version, and this is quite a last minute API change.

> If I read the diff correctly, one deprecated function keeps calling a deprecated action creator, which keeps outputting an action of deprecated type.

Based on your suggestion, the deprecated functions now call their `*FreeformContent*` counterparts. This is simpler, and calling a deprecated block registration function no longer leads to multiple deprecation warnings.